packages/backend/src/apps/stripe/triggers/balance-transaction/index.ts

```diff
@@ -0,0 +1,12 @@
+import defineTrigger from "../../../../helpers/define-trigger";
+import getBalanceTransactions from "./get-balance-transactions";
+
+export default defineTrigger({
+  name: 'New Balance Transactions',
+  key: 'balanceTransaction',
```

Comment:
Could you also replace the key with `newBalanceTransactions` since that is the general convention we want to keep?